index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Israel CHIZUNGU Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <section>
+        <!--  toolbar starts -->
+        <header>
+            <a href="#" class="logo">My Logo</a>
+            <a href="#"><img src="img/hamburger.png" alt="humburger icon" class="hamburger"></a>
+            
+
+        </header>
+        <!--  toolbar ends -->
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove all empty spaces from your code base to make it more readable and organized.